website/frontend/src/components/blocks/SearchForm/SearchForm.tsx

```diff
@@ -1,60 +1,49 @@
-import React, { VFC } from 'react';
+import React from 'react';
+import { Button } from '@gear-js/ui';
 import { Field, Form, Formik } from 'formik';
 import { SearchModel } from 'types/program';
 import { SearchIcon } from 'assets/Icons';
+import searchIcon from 'assets/images/search.svg';
 import { Schema } from './Schema';
-import './SearchForm.scss';
+import styles from './SearchForm.module.scss';
 
 type Props = {
-  handleSearch: (searchQuery: string) => void;
-  handleRemoveQuery: () => void;
+  term: string;
```

Comment:
I don't really like the `term` name here. `query` or something feels more appropriate.

Reply:
In this case we need to change name on the backend too.